01-numpy.html

```diff
@@ -68,17 +68,23 @@ <h2 id="learning-objectives" class="objectives objectives">Learning Objectives</
 <pre class="output"><code>weight in kilograms is now: 57.5</code></pre>
 <p>As the example above shows, we can print several things at once by separating them with commas.</p>
 <p>If we imagine the variable as a sticky note with a name written on it, assignment is like putting the sticky note on a particular value:</p>
-<p><img src="img/python-sticky-note-variables-01.svg" alt="Variables as Sticky Notes" /></p>
+<div class="figure">
+<img src="fig/python-sticky-note-variables-01.svg" alt="Variables as Sticky Notes" /><p class="caption">Variables as Sticky Notes</p>
```

Comment:
On Mon, Jan 19, 2015 at 04:43:05PM -0800, Greg Wilson wrote:
> Is there a way to automatically number figures?

Maybe not yet?  jgm/pandoc#615

It looks like there's an implementation in jgm/pandoc#1298 that's
blocking on a cleaner solution [1].

[1]: https://github.com/jgm/pandoc/pull/1298#issuecomment-52658301